tests/ui/offset-of/offset-of-arg-count.rs

```diff
@@ -3,7 +3,13 @@
 use std::mem::offset_of;
 
 fn main() {
-    offset_of!(NotEnoughArguments); //~ ERROR expected one of
-    offset_of!(NotEnoughArgumentsWithAComma, ); //~ ERROR expected 2 arguments
-    offset_of!(Container, field, too many arguments); //~ ERROR expected 2 arguments
+    offset_of!(NotEnoughArguments); //~ ERROR unexpected end of macro invocation
+    offset_of!(NotEnoughArgumentsWithAComma, ); //~ ERROR unexpected end of macro invocation
+    offset_of!(Container, field, too many arguments); //~ ERROR no rules expected the token `too`
+    offset_of!(S, f); // compiles fine
+    offset_of!(S, f,); // also compiles fine
+    offset_of!(S, f.); //~ ERROR unexpected end of macro invocation
+    offset_of!(S, f.,); //~ ERROR expected identifier
```

Comment:
is it worth also testing `..` as that is a valid syntax in an invalid position?

Reply:
I have added some tests for `..`.